en/editor/extension/scene-script.md

```diff
@@ -0,0 +1,73 @@
+# Call the engine API and project script
+
+In the plugin you can declare a special script file (scene script), the script and the script in the project ( `assets` directory script) has the same environment, that is in this script can call the engine API and other projects Script to achieve:
```

Comment:
In the plugin you can declare a special script file (scene script), the script and the script in the project ( `assets` directory script) has the same environment, that is in this script can call the engine API and other projects Script to achieve:
-->
In a plugin, a special script called a __scene script__ can be created in `assets\`. This special script can call the engine API and other projects scripts to achieve special functionality, including:

Reply:
@slackmoehrle I think the translation of this paragraph should be:

In a plugin, you can declare a special script file called **scene script**, which has the same environment as the scripts in the `assets` directory of the project. That is, in this script you can call the engine API and other project scripts to achieve special functionality, including:

Reply:
@xunyi0 this is good too.